### PR TITLE
Standards Unplugged Lesson Status Dialog shows real data 

### PR DESCRIPTION
**Pull request by Erin007**
[LP-1127](https://codedotorg.atlassian.net/browse/LP-1127)

With this work, we're now using real data to show the unplugged lessons for a given script in the `LessonStatusDialog`. 

Screen Shot 2020-02-03 at 5 11 23 PM

### CODE REVIEW

File: apps/src/templates/sectionProgress/standards/sectionStandardsProgressRedux.js [hunk 1]
```diff
@@ -25,8 +23,25 @@ export default function sectionStandardsProgress(state = initialState, action) {
   return state;
 }
 
-export function getUnpluggedLessonsForScript(script) {
-  return unpluggedLessonList;
+export function getUnpluggedLessonsForScript(state) {
+  const stages =
+    state.sectionProgress.scriptDataByScript[state.scriptSelection.scriptId]
+      .stages;
```
Comment (clareconstantine) [inline apps/src/templates/sectionProgress/standards/sectionStandardsProgressRedux.js]: do we need to be checking that these exist before chaining them? ie, are we sure that `state.sectionProgress.scriptDataByScript[state.scriptSelection.scriptId]` will always exist?

**Review by clareconstantine — APPROVED** [review]

File: dashboard/app/models/stage.rb [hunk 2]
```diff
@@ -61,6 +61,12 @@ def unplugged?
     script_levels.first.oldest_active_level.unplugged?
   end
 
+  def display_as_unplugged
+    script_levels = script.script_levels.select {|sl| sl.stage_id == id}
+    return false unless script_levels.first
+    script_levels.first.oldest_active_level.properties["display_as_unplugged"] == "true" || unplugged?
```
Comment (dmcavoy) [inline dashboard/app/models/stage.rb:70]: We should probably add a note here that this only really works for CSF where we have unplugged levels. There are unplugged lessons in CSD and CSF but they don't use this set up so if we ever want this for those courses we will need to reconsider this

File: dashboard/app/models/stage.rb [hunk 3]
```diff
@@ -61,6 +61,12 @@ def unplugged?
     script_levels.first.oldest_active_level.unplugged?
   end
 
+  def display_as_unplugged
```
Comment (dmcavoy) [inline dashboard/app/models/stage.rb:67]: It would be great to have a test for this function.

**Review by dmcavoy — COMMENTED** [review]
I just love how this is all coming together. I would love to see some tests written for the new logic and then I think this will be good to go

File: dashboard/app/models/stage.rb (hunk 2 above)
Comment (Erin007) [reply dashboard/app/models/stage.rb:70]: How do CSD and CSP handle unplugged status of a lesson?

**Pull request merged**